its/plugin/tests/src/test/java/com/sonar/it/php/PHPIntegrationTest.java

```diff
@@ -65,59 +62,59 @@ public static void startServer() throws IOException, URISyntaxException, Interru
+    assertThat(getProjectMeasure("function_complexity_distribution").getValue()).isEqualTo("1=1236;2=353;4=132;6=71;8=34;10=20;12=36");
+    assertThat(getProjectMeasure("file_complexity_distribution").getValue()).isEqualTo("0=198;5=97;10=78;20=29;30=11;60=10;90=2");
+    assertThat(getProjectMeasureAsDouble("class_complexity_distribution")).isNull();
   }
 
   @Test
   public void fileMetrics() {
     // Size
-    assertThat(getFileMeasure("ncloc").getIntValue()).isEqualTo(41);
-    assertThat(getFileMeasure("lines").getIntValue()).isEqualTo(90);
-    assertThat(getFileMeasure("files").getIntValue()).isEqualTo(1);
-    assertThat(getFileMeasure("classes").getIntValue()).isEqualTo(1);
-    assertThat(getFileMeasure("functions").getIntValue()).isEqualTo(3);
-    assertThat(lineNumbersInDataMeasure(getFileMeasure("ncloc_data").getData())).isEqualTo(lineNumbersInDataMeasure(
+    assertThat(getFileMeasureAsDouble("ncloc")).isEqualTo(41d);
+    assertThat(getFileMeasureAsDouble("lines")).isEqualTo(90d);
+    assertThat(getFileMeasureAsDouble("files")).isEqualTo(1d);
+    assertThat(getFileMeasureAsDouble("classes")).isEqualTo(1d);
+    assertThat(getFileMeasureAsDouble("functions")).isEqualTo(3d);
+    assertThat(lineNumbersInDataMeasure(getFileMeasure("ncloc_data").getValue())).isEqualTo(lineNumbersInDataMeasure(
       "12=1;14=1;21=1;22=1;32=1;33=1;34=1;35=1;37=1;39=1;41=1;42=1;44=1;46=1;47=1;48=1;50=1;52=1;53=1;54=1;56=1;58=1;59=1;60=1;62=1;64=1;66=1;67=1;68=1;70=1;72=1;73=1;75=1;76=1;77=1;78=1;85=1;86=1;87=1;88=1;89=1"));
     if (is_after_sonar_6_2()) {
-      assertThat(lineNumbersInDataMeasure(getFileMeasure("executable_lines_data").getData())).isEqualTo(lineNumbersInDataMeasure(
+      assertThat(lineNumbersInDataMeasure(getFileMeasure("executable_lines_data").getValue())).isEqualTo(lineNumbersInDataMeasure(
         "12=1;14=1;21=1;22=1;32=1;33=1;34=1;35=1;37=1;39=1;41=1;42=1;44=1;46=1;47=1;48=1;50=1;52=1;53=1;54=1;56=1;58=1;59=1;60=1;62=1;64=1;66=1;67=1;68=1;70=1;72=1;73=1;75=1;76=1;77=1;78=1;85=1;86=1;87=1;88=1;89=1"));
-      assertThat(getFileMeasure("lines_to_cover").getIntValue()).isEqualTo(41);
-      assertThat(getFileMeasure("uncovered_lines").getIntValue()).isEqualTo(41);
+      assertThat(getFileMeasureAsDouble("lines_to_cover")).isEqualTo(41d);
+      assertThat(getFileMeasureAsDouble("uncovered_lines")).isEqualTo(41d);
     }
 
     // Comments
-    assertThat(getFileMeasure("comment_lines_density").getValue()).isEqualTo(26.8);
-    assertThat(getFileMeasure("comment_lines").getIntValue()).isEqualTo(15);
-    assertThat(getFileMeasure("public_documented_api_density")).isNull();
-    assertThat(getFileMeasure("public_undocumented_api")).isNull();
+    assertThat(getFileMeasureAsDouble("comment_lines_density")).isEqualTo(26.8);
+    assertThat(getFileMeasureAsDouble("comment_lines")).isEqualTo(15d);
+    assertThat(getFileMeasureAsDouble("public_documented_api_density")).isEqualTo(100);
```

Comment:
@pynicolas Because this metric is using a best value (https://github.com/SonarSource/sonarqube/blob/master/sonar-plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java#L309) => it means that when no measure are computed, the best value is returned.
The old WS api/resources/index was not handling best values, but the one is handling it, which is explaining this new behavior in ITs.